es6-destructuring/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ES6-Destructuring</title>
+</head>
+<body>
+  <script>
+  const book1 = {
+    title: 'Goodnight Punpun',
+    author: 'Inio Asano',
+    libraryID: 3353
+};
+
+const book2 = {
+    title: 'Les Fleurs du mal',
+    author: 'Charles Baudelaire',
+    libraryID: 2345
+};
+
+const library = [
+    {
+      title: 'The Road Ahead',
+      suthor: 'Bill Gates',
+      libraryID: 1254
+    },
+    {
+      title: 'Walter Isaacson',
+      author: 'Steve Jobs',
+      libraryID: 4264
+    },
+    {
+      title: 'Mockingjay: The Final Book of The Hunger Games',
+      author: 'Suzanne Colling',
+      libraryID: '3245'
+    },
+    {
+      title: "Dead Dead Demon's De De De De Destruction",
+      author: 'Inio Asano',
+      libraryID: 1233
+    }
+];
+
+const { title: bookTitle1, author: authorName1, libraryID: libID1 } = book1;
+const { title: bookTitle2, author: authorName2, libraryID: libID2 } = book2;
+const [book3, book4, book5] = library;
+const [, book6] = library;
```

Comment:
`book6` should be the _last_ book, not the 2nd one:

---

![image](https://user-images.githubusercontent.com/16143771/156293571-6ccd6e99-459b-4604-9673-d626451e5be2.png)

---

![image](https://user-images.githubusercontent.com/16143771/156293633-1fd3e25b-6ad4-4be3-95c6-a7161227c85d.png)

Reply:
![Screenshot 2022-03-01 204354](https://user-images.githubusercontent.com/25161542/156296898-cbd95ebd-6999-4e50-a67a-7d9d9ebedefa.png)